### PR TITLE
Add cwl_utils.parser.latest

### PR DESCRIPTION
**Pull request by tom-tan**
This request adds `cwl_utils.parser.latest` module for the latest CWL version.
It just import all the symbols in the module for the latest CWL version (i.e., `cwl_utils.paresr.cwl_v1_2`).

Related: https://github.com/common-workflow-language/cwl-utils/pull/102#discussion_r730702832

### CODE REVIEW

**Comment by mr-c** [comment]
Might be better to do `import .cwl_v1_2 as latest` in `parsers/__init__.py` instead?

**Comment by tom-tan** [comment]
> Might be better to do `import .cwl_v1_2 as latest` in `parsers/__init__.py` instead?

It causes the following problem:

```python
import cwl_utils.parser as parser

...
cwl_obj = parser.latest.load_document_by_yaml(yaml) # it seems there is `cwl_utils.parser.latest` module but...
import cwl_utils.parser.latest as cwl_latest  # it does not work!
```

It is better to add `from . import .cwl_v1_2 as latest` in `parsers/__init__.py` in addition to provide `cwl_utils.parser.latest` moudle.

**Review by mr-c — APPROVED** [review]

**Comment by mr-c** [comment]
Okay, we'll leave this as is. Can we get a test?

**Comment by tom-tan** [comment]
Is it acceptable to add a test that loads the latest (v1.2) document with `cwl_utils.parser.latest.load_document_by_yaml`, for example?

Of course we can easily add such a test but it is not straightforward way for mea little bit fragile.
We have to do the following when the new version of CWL spec is released:
1. add a document of the newly released version of CWL to `testdata/` and
2. fix the test to load the document that is added in 1.
For me, it seems to be a trickier than we want to do.

What we want to do is that `cwl_utils.parser.latest`

**Comment by tom-tan** [comment]
Is it acceptable to add a test that loads the latest (v1.2) document with `cwl_utils.parser.latest.load_document_by_yaml`, for example?
In this case, we check that `latest` is same as `cwl_v1_2` by trying to load the latest version of CWL document.

Ideally it would be nice if each parser module provides a constant such as `parser_info` and if we just add a test as follows:
```python
from cwl_utils.parser.latest import parser_info
...
assert parser_info == "cwl-v1.2"
```

**Comment by mr-c** [comment]
> Is it acceptable to add a test that loads the latest (v1.2) document with `cwl_utils.parser.latest.load_document_by_yaml`, for example?
> In this case, we check that `latest` is same as `cwl_v1_2` by trying to load the latest version of CWL document.

Sure.

> Ideally it would be nice if each parser module provides a constant such as `parser_info` 

Good idea, I've opened https://github.com/common-workflow-language/schema_salad/issues/473 to keep track of that.

File: cwl_utils/parser/__init__.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 from . import cwl_v1_0 as cwl_v1_0
 from . import cwl_v1_1 as cwl_v1_1
 from . import cwl_v1_2 as cwl_v1_2
+from . import latest as latest
```
Comment (mr-c) [inline cwl_utils/parser/__init__.py:8]:
```suggestion
```

Unless you are going to add code as part of this PR that uses `latest` in `cwl_utils/parser/__init__.py`

**Comment by tom-tan** [comment]
Done!
I will fix the test once common-workflow-language/schema_salad#473 is solved but it is out of scope of this request.

**Review by mr-c — APPROVED** [review]

**Pull request merged**